### DISCUSSION TITLE
Alexa ha dejado de funcionar

### DISCUSSION THREAD

**Discussion by Toskilabis** [BimmerGestalt/AAIdrive#544] [Q&A] [answered]
Hola desde haces unos días Alexa ha dejado de funcionar con la opción Push and Talk en ID6. Ha funcionado perfectamente con Android y de un día para otro cuando presionas el comando de voz se oye un mensaje : " Esta opción no está disponible actualmente".
La suscripción Conected Drive esta activa, he vuelto a desinstalar e instalar las apps My BMW, AutoIdrive y Alexa vinculando mi cuenta Amazon de nuevo en My BMW y no consigo hacerlo funcionar mediante comando de voz
Alguna opinión al respecto?
Gracias

**Comment by hufman** [comment]
Hmmm I haven't heard of this problem, though I haven't tried this feature often. Is your phone fully connected to the car? In IDrive5/6, I think the Alexa app requires the same Apps tunnel that AAIdrive uses, is that working properly? The MyBMW app should show a status icon about whether the Apps are connected, as another clue.

**Reply by Toskilabis** [reply]
The phone is a Xiaomi and it is perfectly integrated, Spotify works, Reading messages, Amazon Music, covers, etc. etc. All these functions and the My Bmw app appear in the list of apps in Connected Drive.
The funny thing is that I have been able to give Alexa voice commands for a while, but now my Bmw says that this option is not available. I must manage Alexa through the Smartphone as I have been doing with Spotify.
Something must have changed but I can't find the solution. Pressing the push and talk button for two seconds doesn't work Ok Google either. It does absolutely nothing. Thanks for your answer

**Comment by Toskilabis** [comment]
Hello again, it seems that Alexa works again after pressing the voice command for two seconds. It is necessary to have the Alexa application installed on the Smartphone. Just install the version with the Mapbox plugin and it worked again.

**Reply by hufman** [reply]
I'm happy to hear that!

**Comment by MA200677** [comment]
Hi! Have the same problems since some time. What you mean with "install the version with the Mapbox plugin"?

**Comment by Toskilabis** [comment]
You can find this version in the explanation of the article on the main page of the project. It belongs to a separate build.

**Comment by MA200677** [comment]
The alternative version of AAIDrive could not solve the problem. Alexa still don´t work. Only deinstallation of AAIDrive brings back the functionality of Alexa. May you check, whats the problem?

**Reply by hufman** [reply]
How are you using Alexa when AAIdrive is uninstalled? The native BMW Alexa integration should show up on the Vehicle Apps section, and uses a separate protocol to your phone. The AAIdrive Alexa app shows under the Phone Apps section, and can not affect the Vehicle Apps section. AAIdrive also has no control over the steering wheel voice assistant button.

**Comment by MA200677** [comment]
I always use the native Alexa implementation via the control button on the steering wheel. Never using it with the AAI Drive implementation (This only starts the voice voice regocnition of the mobile app.). But, if the AAI Drive App is installed, the native integration of Alexa tell me, that ist is not available. When AAI Drive is uninstalled, the BMW Alexa integration works fine.

**Reply by hufman** [reply]
Very peculiar! There shouldn't be any interaction between them, and (after I set up the Amazon account link in the Mini app) it works whether AAIdrive is running or not, even if it stays installed.

**Reply by Toskilabis** [reply]
It must work like this on Apple. On my Android Phone if I don't have AIdrive Alexa installed it doesn't work for me. The problem is that it fails again after a few days and I have to uninstall the My BMW and Aidrive app again. I reinstall the two apps again and everything is ok. I am unable to identify what the problem is.

**Reply by MA200677** [reply]
I think I found the problem. It has something to do with the myBMW App. You dont´t need to deinstall AAI Drive or the myBMW App, just disconnect the Amazon Account and connect ist again, while you are connected with the car. I will send a troubleticket to BMW.

**Reply by Toskilabis** [reply]
You're right. After reinstalling Aidrive and My Bmw, Alexa failed again. So I disconnected and reconnected my Amazon account in the My BMW app and Alexa worked again without uninstalling anything. Please comment here the solution offered by Bmw to your request.
Thanks

**Comment by Toskilabis** [comment, marked as answer]
New Update of the My Bmw App and now Alexa does not connect to the Internet. Adds to the problem of having to disconnect and reconnect the Amazon account every time you get in the car. I confirm that Android requires Aiidrive installed for Alexa to show signs of life. The My BMW App a real mess getting worse.
I have also reported the problem to BMW. Waiting answer

**Comment by Toskilabis** [comment]
Waiting for a solution from Bmw of all the problems that the latest update of its app is generating, I have uninstalled the latest version and reinstalled version 1.4 from here:

https://apkaio.com/app/de.bmw.connected.mobile20.row

Alexa works correctly again from the voice command. Make sure you skip updating the app.

**Reply by MA200677** [reply]
I am still waiting for an answer from BMW as well. But I found something, that might be the soluion. Ma Android phone told me for weeks that the myBMW App has a lot of background activity and if i wish to get it in standby. I tried it. When I now enter my car, the app interface still don´t connect by itself to the car. I have to start the myBMW app first, then the app interface will be activated and tadaa, the Alexa integration works every time.

**Reply by Toskilabis** [reply]
> A la espera de una solución por parte de Bmw de todos los problemas que está generando la última actualización de su app, he desinstalado la última versión y reinstalado la versión 1.4 desde aquí:
> 
> https://apkaio.com/app/de.bmw.connected.mobile20.row
> 
> Alexa vuelve a funcionar correctamente desde el comando de voz. Asegúrate de omitir la actualización de la aplicación.

In this link at the end of the article you will find older versions of the My Bmw app. Reinstall any 1.4 version and disable automatic update. Everything works correctly again and the app connects itself to the car.

**Comment by Toskilabis** [comment]
New web with all versions of the My BMW app to date

https://apkcombo.com/en/my-bmw/de.bmw.connected.mobile20.row/old-versions

**Comment by Alberto5624** [comment]
En BMW dicen que están actualizando los servidores y por eso no funciona. Eso me parece raro, si es verdad que desconectando la cuenta de Amazon y volviendo a conectarla funciona pero eso no deja de ser una solución parcial.
